blockapi/api/mirror.py

```
@@ -0,0 +1,302 @@
+from decimal import Decimal
+
+from blockapi.services import BlockchainAPI
+
+
+class MirrorApi(BlockchainAPI):
+
+
+    symbol = 'LP'
+    base_url = 'https://tequila-lcd.terra.dev'
+    rate_limit = 0.5
+    coef = Decimal(1e-6)
+    max_items_per_page = 100
+    page_offset_step = 1
+
+    supported_requests = {
+        'get_mirror_pool_balance':  '/wasm/contracts/{lp_token_contract}/store' 
+                                    '?query_msg={{ "balance" : {{  "address" : "{address}" }} }}',
+        'get_mirror_pool':          '/wasm/contracts/{pair_contract}/store'
+                                    '?query_msg={{ "pool": {{ }} }}'
+    }
+
+    symbols = {
+        'uluna': 'LUNA',
+        'ukrw': 'KRT',
+        'usdr': 'SDT',
+        'uusd': 'UST',
+        'umnt': 'MNT'
+    }
+
+    liqudity_pool_suffix = "LP"
+
+
+    def get_balance(self):
```

This behaviour is not correct - if you do now want to support `get_balance` method, you should raise an exception, not return an empty string.
```
def get_balance(self):
    raise NotImplementedError()
```

---

Definitely. Or better return get_lp_balance();